### PR TITLE
WIP: add sql script for backup reasons

### PR DESCRIPTION
**Pull request by Laurin-dev**

### CODE REVIEW

File: Movie-Db-CSS.sql [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+-- Table structure for table `Movies`
+--
+
+DROP TABLE IF EXISTS `Movies`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `Movies` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `title` varchar(45) DEFAULT NULL,
+  `year` varchar(45) DEFAULT NULL,
```
Comment (fabianmueller7) [inline Movie-Db-CSS.sql:28]: Date?

File: Movie-Db-CSS.sql [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+--
+-- Table structure for table `Movies`
+--
+
+DROP TABLE IF EXISTS `Movies`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `Movies` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `title` varchar(45) DEFAULT NULL,
```
Comment (fabianmueller7) [inline Movie-Db-CSS.sql:27]: There are some realy long movie names, but thats okay.

**Review by fabianmueller7 — CHANGES_REQUESTED** [review]
Have a look at the comments please.

**Comment by DeNic0la** [comment]
There is a Better, Simpler SQL script already added. I Dont see the Value in this

**Pull request closed without merging**